crowdsec-docs/unversioned/getting_started/post_installation/acquisition.mdx

```diff
@@ -0,0 +1,252 @@
+:::
+
+```bash
+cscli collections add <collection_name>
+```
+
+Once the collection has been downloaded we need to add a new acquisition to the configuration.
+
+:::info
+From version `1.5.0` we have added the ability to add acquisitions via the `acquis.d` directory. This allows you to add acquisitions without modifying the main configuration file.
+:::
+
+### Create the directory if it does not exist
+
+<Tabs
+  defaultValue="nix"
+  groupId="console-operating-systems"
+  values={[
+    {label: 'Linux/Freebsd', value: 'nix'},
+    {label: 'Windows', value: 'windows'},
+  ]}>
+  <TabItem value="nix">
+    <CodeBlock className="language-bash">sudo mkdir -p /etc/crowdsec/acquis.d</CodeBlock>
+  </TabItem>
+  <TabItem value="windows">
+        <Tabs
+    defaultValue="powershell"
+    groupId="windows-terminal"
+    values={[
+      {label: 'Powershell', value: 'powershell'},
+      {label: 'CMD', value: 'terminal'},
+    ]}>
+      <TabItem value="powershell">
+          ```bash
+          New-Item -ItemType Directory -Force -Path C:\ProgramData\CrowdSec\Config\acquis.d\
+          ```
+      </TabItem>
+      <TabItem value="terminal">
+          ```bash
+          mkdir "C:\ProgramData\CrowdSec\Config\acquis.d\"
+          ```
+      </TabItem>
+    </Tabs>
+  </TabItem>
+</Tabs>
+
+### Create the acquisition file
+
+<Tabs
+  defaultValue="nix"
+  groupId="console-operating-systems"
+  values={[
+    {label: 'Linux/Freebsd', value: 'nix'},
+    {label: 'Windows', value: 'windows'},
+  ]}>
+  <TabItem value="nix">
+    ```bash
+    sudo touch /etc/crowdsec/acquis.d/<collection_name>.yaml
+    ```
+  </TabItem>
+  <TabItem value="windows">
+        <Tabs
+    defaultValue="powershell"
+    groupId="windows-terminal"
+    values={[
+      {label: 'Powershell', value: 'powershell'},
+      {label: 'CMD', value: 'terminal'},
+    ]}>
+      <TabItem value="powershell">
+          ```bash
+          New-Item -ItemType File -Force -Path C:\ProgramData\CrowdSec\Config\acquis.d\<collection_name>.yaml
+          ```
+      </TabItem>
+      <TabItem value="terminal">
+          ```bash
+          .>C:\ProgramData\CrowdSec\Config\acquis.d\<collection_name>.yaml 2>NUL
+          ```
+      </TabItem>
+    </Tabs>
+  </TabItem>
+</Tabs>
+
+### Add the following contents to the file
+
+```yaml title="<collection_name>.yaml"
+filesnames:
+  - "/path/to/your/file.log" ## Single file
+  - "/path/to/your/files*" ## Wildcard support
+labels:
+  type: "<type of logs>" ## Type defined in the parser
+```
+
+:::tip
+You can find an example acquisition within the collection shown on the [hub](https://hub.crowdsec.net/), this will help you understand what files you need to monitor and what labels you need to add.
+:::
+
+Once you have added the acquisitions you can test the configuration and restart the service.
```

Comment:
We might suggest them to go back to the `cscli collections inspect` or `cscli metrics` part to check things are OK.